tests/unit/auth/v4/queryAuthCheck.js

```diff
@@ -225,4 +225,18 @@ describe('v4 queryAuthCheck', () => {
         assert.strictEqual(res.params.version, 4);
         done();
     });
+
+    it('should successfully return no error if proxy_path header is added',
```

Comment:
Will there ever be a case where proxy_path header is added but invalid?

Reply:
Good catch!! Thanks, I addressed your comment.